packages/cli/package.json

```diff
@@ -41,6 +41,9 @@
   "bin": {
     "code-pushup": "./src/index.js"
   },
+  "engines": {
+    "node": ">=20"
+  },
```

Comment:
Should we add this to all our packages? :thinking:

Reply:
> I assume you always need to use CLI to run anything

That's true. Users also import plugins directly and may import from utils as well (when building their own plugins, typically), but in the end they'll run it through the CLI.